service_container/autowiring.rst

```diff
@@ -363,9 +363,37 @@ If you register this as a service, you now have *two* services that implement th
 which one to use. Remember, autowiring isn't magic; it looks for a service
 whose id matches the type-hint. So you need to choose one by creating an alias
 from the type to the correct service id (see :ref:`autowiring-interface-alias`).
+Additionally, you can define several named aliases if you want to use
+one implementation in some cases, and another implementation in some
+other cases.
 
-If you want ``Rot13Transformer`` to be the service that's used for autowiring, create
-that alias:
+If you want ``Rot13Transformer`` to be the service that's used for
+autowiring by default, and use ``UppercaseTransformer`` in some special
+cases, you can create a simple alias from the interface to
```

Comment:
I would put `normal` instead of `simple`

Reply:
Or just nothing: `In some special cases, you can create an alias from the interface (...)`. An alias is straightforward enough 👍